src/App.scss

```diff
@@ -1,39 +1,110 @@
+@import "./utils/Normalize.scss";
+@import "./utils/Reset.scss";
+@import "./utils/Mixins.scss";
+@import "./utils/variables.scss";
+
 iframe {
   display: none;
 }
 
-html {
-  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+.App {
+  @include flex-column;
+}
+
+.header {
+  background-color: $-color-background;
```

Comment:
Consider using a more descriptive property name instead of `background-color`. 🤔